wasmer/global.go

```diff
@@ -33,6 +35,40 @@ func (self *Global) ownedBy() interface{} {
 	return self._ownedBy
 }
 
+func (self *Global) Set(value interface{}, kind ValueKind) error {
+	value, err := fromGoValue(value, kind)
+	wasmValue := value.(C.wasm_val_t)
```

Comment:
`value` is already a `C.wasm_val_t`, you don't need to assert the type here, right?